pages/dashboard.html

```diff
@@ -5,10 +5,267 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.2/font/bootstrap-icons.css">
+    <link rel="stylesheet" href="../style/style.css">
+
+    <title>Dashboard</title>
 </head>
 
 <body>
+    <header>
+        <a class="logo" href="#">my<span>Bank</span></a>
+
+        <img class="img"
+            src="../library/profile.webp"
+            alt="">
+        <i class="bi bi-box-arrow-right , iconLogOut"></i>
```

Comment:
let this icon disappear if the user is not logged in

Reply:
Done. I have already fixed the code, please check

Reply:
Done. I have already fixed the code, please check